tools/c7n_azure/c7n_azure/resources/aci.py

```diff
@@ -42,6 +42,7 @@ class resource_type(ArmResourceManager.resource_type):
         default_report_fields = (
             'name',
             'location',
-            'resourceGroup'
+            'resourceGroup',
+            'properties.containers[].name'

```

Comment:
What if ACI hosts 10+ containers? Report won't be readable..

Reply:
Yes this would be ugly. Thought it was still important but we can drop

Reply:
You can do container count if you want